static/bidder-info/smartx.yaml

```diff
@@ -0,0 +1,12 @@
+endpoint: "https://bid.smartclip.net/bid/1005"
+maintainer:
+  email: "bidding@smartclip.tv"
```

Comment:
@fkoch-sc  PBS team has sent mail for maintainer email verification.  Requesting to respond with "received" message to confirm the email address.

Reply:
Received.